src/components/lib/posix/posix.c

```diff
@@ -34,46 +39,29 @@ libc_syscall_override(cos_syscall_t fn, int syscall_num)
 int
 cos_open(const char *pathname, int flags, int mode)
 {
-	/* mode param is only for O_CREAT in flags */
-	td_t td;
-	long evt;
-	evt = evt_split(cos_spd_id(), 0, 0);
-	assert(evt > 0);
-	td = tsplit(cos_spd_id(), td_root, (char *)pathname, strlen(pathname), TOR_ALL, evt);
-
-	if (td <= 0) {
-		printc("open() failed!\n");
-		assert(0);
-	}
-
-	return td + 3; /* fd number adjust for reserved fd 0, 1 and 2 */
+	printc("open not implemented\n");
+	return 0;
```

Comment:
Should these "not implemented" methods fault? It feels like you want to fail fast if these syscalls aren't implemented.